src/biblebot/bot.py

```
@@ -1018,49 +960,18 @@ async def on_room_message(self, room: MatrixRoom, event: RoomMessageText):
-                    )
-                    break
+            match = detect_trigger(
+                body=event.body,
+                default_translation=self.default_translation,
+            )
 
-            if passage:
+            if match:
+                logger.info(
+                    f"Detected Bible reference ({match.source.value}): {match.passage} "
+                    f"({match.translation}) in room {room.room_id}"
```

Ruff (0.15.10)

[warning] Logging statement uses f-string

(G004)